### PR TITLE
Quiz logic

### PR DESCRIPTION
**Pull request by bryamrr**

### CODE REVIEW

File: swift-quiz.playground/Contents.swift [hunk 1]
```diff
@@ -1,5 +1,48 @@
 //: Playground - noun: a place where people can play
 
-import UIKit
+class Quiz {
+    let result:Int = 0
```
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: result, must not be stored as a variable, take a more functional approach

File: swift-quiz.playground/Contents.swift [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+    
+    let arrQuestions:[Question] = []
+    
+    func addQuestions(questions: [AnyObject]) {
+        // Create Question objects and add to arrQuestions
+    }
+    
+    func calc(arrAnwsers: [AnyObject]) {
```
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: Rename tu evaluate()... return the result

File: swift-quiz.playground/Contents.swift [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+        // Calc quiz result
+    }
+    
+    func showResults() {
```
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: Remove

File: swift-quiz.playground/Contents.swift [hunk 4]
```diff
@@ -1,5 +1,48 @@
 //: Playground - noun: a place where people can play
 
-import UIKit
+class Quiz {
+    let result:Int = 0
+    
+    let arrQuestions:[Question] = []
```
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: Don't use "arr" prefix

File: swift-quiz.playground/Contents.swift [hunk 5]
```diff
@@ -1,5 +1,48 @@
...
+    let multiply:Bool = false // According to this, score will just add or will multiply
+    let arrchoices:[Choice] = []
+    
+    func addChoices(choices: [AnyObject]) {
+        // Create choice and add to arrChoices
+    }
+    
+}
+
+class Choice {
```
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: Rename "Answer"

File: swift-quiz.playground/Contents.swift (hunk 1 above)
Comment (bryamrr) [inline swift-quiz.playground/Contents.swift]: What do you mean? I just have to return it inside the function?

File: swift-quiz.playground/Contents.swift [hunk 6]
```diff
@@ -1,5 +1,48 @@
 //: Playground - noun: a place where people can play
 
-import UIKit
+class Quiz {
+    let result:Int = 0
+    
+    let arrQuestions:[Question] = []
+    
+    func addQuestions(questions: [AnyObject]) {
```
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: I think Quiz object must have a constructor with 1 json string containing the entire data structure.

File: swift-quiz.playground/Contents.swift (hunk 1 above)
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: yep

Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: always prefer stateless code

File: swift-quiz.playground/Contents.swift [hunk 7]
```diff
@@ -1,5 +1,68 @@
...
 
-import UIKit
+class Quiz {
+    let questions:[Question] = []
+    
+    func addQuestions(questions: [AnyObject]) {
+        // Create Question objects and add to arrQuestions
+    }
+    
+    func evaluate(arrAnwsers: [AnyObject]) {
```
Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: Typo "Anwsers"

Comment (lshimokawa) [inline swift-quiz.playground/Contents.swift]: Missing return type

**Pull request merged**